styleguide/components/41-organisms/footer/README.md

```diff
@@ -1,9 +1,47 @@
 # Footer
 
+## When to use this component
+
+All websites and all pages must include the footer.
+
+The footer should be used **as is**. The elements, style, layout and content in the footer **cannot be changed in anyway**, with only a few exceptions when used on separate websites.
```

Comment:
I find this kind of vague advice; saying "You can't change it in any way, but you can if you want to make an exception" 🤔

Maybe you could specify in which cases you **are** allowed to change it?

Reply:
I made an update to this documentation. Could you check if it is more clear now?